platform/windows/native_menu_windows.cpp

```diff
@@ -177,6 +177,16 @@ void NativeMenuWindows::popup(const RID &p_rid, const Vector2i &p_position) {
 	}
 	SetForegroundWindow(hwnd);
 	TrackPopupMenuEx(md->menu, flags, p_position.x, p_position.y, hwnd, nullptr);
+
+	if (md && md->close_cb.is_valid()) {
```

Comment:
```suggestion
	if (md->close_cb.is_valid()) {
```

It's already checked.

Reply:
Fixed, forget that in previous line we are using it.